Libraries/PeriphDrivers/Source/LP/lp_me14.c

```diff
@@ -427,6 +427,18 @@ int MXC_LP_BandgapIsOn(void)
     return (MXC_PWRSEQ->lpcn & MXC_F_PWRSEQ_LPCN_BGOFF);
 }
 
+void MXC_LP_EnableUSBWakeup(void)
+{
+    MXC_MCR->ctrl &= ~MXC_F_MCR_CTRL_USBSWEN_N;
+    MXC_PWRSEQ->lppwen |= MXC_F_PWRSEQ_LPPWEN_USBVBUSWKEN;
```

Comment:
Do we also need to set/clear the USB line state change wakeup enable field in these functions?

Reply:
This is needed. You can refer lp_me10.c. This bit is sampled entering deepsleep or backup mode.

Reply:
Yes, but is it also necessary to set/clear MXC_F_PWRSEQ_LPPWEN_USBLSWKEN?

Like,
 ```
MXC_PWRSEQ->lppwen |= MXC_F_PWRSEQ_LPPWEN_USBVBUSWKEN | MXC_F_PWRSEQ_LPPWEN_USBLSWKEN;
```

Reply:
Added wakeup from VBUS and USB line state change bits.
Thanks for pointing out.